app/code/core/Mage/Catalog/Model/Product/Attribute/Source/Countryofmanufacture.php

```diff
@@ -45,10 +45,15 @@ public function getAllOptions()
         if (Mage::app()->useCache('config') && $cache = Mage::app()->loadCache($cacheKey)) {
             $options = unserialize($cache);
         } else {
-            $collection = Mage::getModel('directory/country')->getResourceCollection();
+            if (Mage::app()->getStore()->isAdmin()) {
+                $collection = Mage::getModel('directory/country')->getResourceCollection();
+            } else {
+                $collection = Mage::getModel('directory/country')->getResourceCollection()->loadByStore();
+            }
             $options = $collection->toOptionArray();
+
             if (Mage::app()->useCache('config')) {
-                Mage::app()->saveCache(serialize($options), $cacheKey, array('config'));
+                Mage::app()->saveCache(serialize($options), $cacheKey, ['config']);
```

Comment:
If this is a purely cosmetic change I would recommend removing it. Same for the introduced new line. Less changes = better.

Reply:
ok thanx!